app/views/trustee_details.scala.html

```diff
@@ -33,65 +33,53 @@
+                        <label class="@if(trusteeDetails(trusteeDetailsFields.country).value.contains(ersUtil.DEFAULT_COUNTRY) || trusteeDetails(trusteeDetailsFields.country).value.isEmpty) {selected}" for="uk-radio-button">@messages("ers_trustee_details.uk")</label>
+                    </div>
+                    <div class="multiple-choice">
+                        <input id="overseas-radio-button" name="radio-inline-group" value="2" type="radio" @if(!trusteeDetails(trusteeDetailsFields.country).value.contains(ersUtil.DEFAULT_COUNTRY) && trusteeDetails(trusteeDetailsFields.country).value.isDefined) {checked="checked"}>
+                        <label class="@if(!trusteeDetails(trusteeDetailsFields.country).value.contains(ersUtil.DEFAULT_COUNTRY) && trusteeDetails(trusteeDetailsFields.country).value.isDefined) {selected}" for="overseas-radio-button">@messages("ers_trustee_details.overseas")</label>
+                    </div>
                 </div>
             </fieldset>
         </div>
-        <div class="form-group @if(trusteeDetails.error(trusteeDetailsFields.name).isDefined) {error}">
-            <label for="name" class="form-label" id="nameLbl">
-                <span>@messages("ers_trustee_details.name")</span>
-                @if(trusteeDetails.error(trusteeDetailsFields.name).isDefined) {<span class="error-message">@trusteeDetails.errors(trusteeDetailsFields.name).head.message</span>}
-            </label>
-            <input id="name" type="text" class="form-control" name="name" value="@trusteeDetails(trusteeDetailsFields.name).value">
+        <div class='form-group'>
+            @helpers.input(trusteeDetails(trusteeDetailsFields.name), '_label -> messages("ers_trustee_details.name"), '_inputClass -> "form-control form-control--block" )
         </div>
-        <div class="form-group @if(trusteeDetails.error(trusteeDetailsFields.addressLine1).isDefined) {error}">
-            <label for="addressLine1" class="form-label" id="addressLine1Lbl">
+        <div class="form-group @if(trusteeDetails.error(trusteeDetailsFields.addressLine1).isDefined) {form-field--error}">
+            <label for="addressLine1" class="form-label">
                 <span>@messages("ers_trustee_details.address")</span>
                 <span class="visuallyhidden">@messages("ers_trustee_details.addressLine1")</span>
-                @if(trusteeDetails.error(trusteeDetailsFields.addressLine1).isDefined) {<span class="error-message">@trusteeDetails.errors(trusteeDetailsFields.addressLine1).head.message</span>}
+                @if(trusteeDetails.error(trusteeDetailsFields.addressLine1).isDefined) {<span class="error-notification" role="tooltip">@trusteeDetails.errors(trusteeDetailsFields.addressLine1).head.message</span>}
```

Comment:
These if statements have been removed from each field apart from this one (addressLine1) in the group, is this what we want?

Reply:
the other field adresses have been swapped to use play ui templates where due to the hint text being hidden this has had to be done manually annoyingly